routingpy/routers/valhalla.py

```diff
@@ -519,7 +519,7 @@ def parse_isochrone_json(response, intervals, locations, interval_type):
                     Isochrone(
                         geometry=feature["geometry"]["coordinates"],
                         interval=intervals[idx],
-                        center=locations,
+                        center=locations[0],
```

Comment:
this is fine for now. but actually, the way we do it here is not really with valhalla. it's never returning more than one isochrone, even with 5 input locations. basically it does a geometric union of the isochrones from multiple locations, so we'll always have only one `Isochrone` in `Isochrones`. which makes the `center` rather meaningless. but again, this is surely better than it was before.

 for valhalla we would need to do a bit more to support actual center coords. we return them conditionally with `show_locations`. in fact we could remove `show_locations` from the input, always append it for isochrones and parse the `MultiPoint` snapped s